services/notification/notifiers/comment/__init__.py

```diff
@@ -38,7 +37,6 @@ class CommentNotifier(MessageMixin, AbstractBaseNotifier):
     notify_conditions: List[NotifyCondition] = [
         ComparisonHasPull,
         PullRequestInProvider,
-        PullRequestOpen,
```

Comment:
Could notifying on closed pull requests lead to an unforeseen increase in rate limit?

Reply:
yeah, that's one of the drawbacks, but i think it's at most an extra api call per PR that's closed.

Reply:
sounds good, I think we can go for it and if we see a spike we likely know why. We can also, and likely, make it a yaml config option to specify commenting post merge